src/components/shared/ErrorImporting/index.tsx

```diff
@@ -0,0 +1,48 @@
+import FullPageSpinner from 'components/fullPage/Spinner';
+import { useEffect, useState } from 'react';
+import {
+    failedToLazyLoad,
+    logRocketConsole,
+    logRocketEvent,
+} from 'services/shared';
+import { CustomEvents } from 'services/types';
+import Error from 'components/shared/Error';
+import { FallbackProps } from 'react-error-boundary';
+import { getWithExpiry, setWithExpiry } from './shared';
+
+export function ErrorImporting({ error }: FallbackProps) {
+    const [stopTrying, setStopTrying] = useState(false);
+
+    // TODO (error importing) get rid of this logging.
+    //  Keep an eye on the error so we know the list for
+    //  failedToLazyLoad is valid.
+    logRocketConsole('Error Importing Error', error);
```

Comment:
Is the repetition of the word _error_ intentional?

Reply:
yeah - I'll add a colon... this is actually logging the `name` of the entity and then saying there is an error.